docs/manifest.json

```diff
@@ -1164,6 +1164,11 @@
           "title": "Scanning Coder Workspaces with JFrog Xray",
           "description": "Integrate Coder with JFrog Xray",
           "path": "./guides/xray-integration.md"
+        },
+        {
+          "title": "Cloning Git Repositories",
+          "description": "Integrate Coder with Git",
```

Comment:
The description may be a little vague for the docs search.

"Automatically clone Git repositories into your workspace."